server/src/server.js

```diff
@@ -35,9 +35,7 @@ server.get('/users/:userId', (req, res) => {
   User.findById(userId)
     .then((user) => res.json(user))
     .catch((error) => {
-      console.log(
-        'User not yet registered. Please consider creating a profile'
-      );
+      console.error(error.message);
```

Comment:
Good job. :) Much better than console-logs

Reply:
True that :)